microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -19,6 +19,18 @@ On Windows 11, widgets appear in the widgets board which you open from the left
 In Windows 11, Progressive Web Apps (PWAs) can define widgets, update them, and handle user interactions within them.
 
 
+<!-- ------------------------------ -->
+#### Requires developing a PWA as a widget
+
+For a PWA-driven widget, you build the widget experience by using the web app manifest, Adaptive Card templates, and JavaScript, rather than putting an existing PWA into a widget.  A PWA-driven widget is a separate experience, distinct from PWAs in the sidebar, which takes the entire PWA and puts it in the sidebar.
+
+No C++/C# code is required, to build a PWA-backed widget and ship through the Microsoft store.  Once you have produced your widget and can successfully install and run your widget from a public endpoint (the tool doesn't work with localhost), you can package your app using PWABuilder.com and ship the app to the Microsoft store without any additional code.
```

Comment:
1. Is this comma intentional?
2. It's not treating PWABuilder.com as a link which might confuse folks, plus linking to the site helps them find additional info about shipping their app
```suggestion
No C++/C# code is required to build a PWA-backed widget and ship through the Microsoft store.  Once you have produced your widget and can successfully install and run your widget from a public endpoint (the tool doesn't work with localhost), you can package your app using [PWABuilder.com](https://www.pwabuilder.com/) and ship the app to the Microsoft store without any additional code.
```

Reply:
flipped the comma sentence phrases.  linked.